concurrent_test.go

```diff
@@ -24,6 +24,22 @@ import (
 	"go.etcd.io/bbolt/internal/common"
 )
 
+type operationInterval struct {
+	minInterval time.Duration
+	maxInterval time.Duration
+}
```

Comment:
In context it would be simpler
```suggestion
type interval struct {
	start time.Duration
	end time.Duration
}
```

Reply:
done.